Model/DutyCalculator.php

```diff
@@ -132,7 +132,7 @@ public function getDutyandTax($cartId, $shippingCharge, $shippingMethodCode, $sh
                 return $this->response;
             }
            
-            if (empty($address->getCountryId()) || empty($address->getRegionCode())) {
```

Comment:
I don't understand how this works, if the country is empty then we got the duties and taxes? And this is an OR statement, so it should have worked before right?

Reply:
Clarified with Ervin, if the country is empty (which it won't ever be) then the duties would be reset to 0 and we would never call DHL. That's what this block is doing